package.json

```diff
@@ -589,46 +591,43 @@
     {
       "id": "Razor",
       "description": "Razor Language Server (Windows / x64)",
-      "url": "https://download.visualstudio.microsoft.com/download/pr/fddc6f7e-083a-44bd-9a8d-200a4146b13b/c51fc9fae3f6fa2b1992df6dd16f50d2/razorlanguageserver-win-x64-7.0.0-preview.23301.2.zip",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/51d2bf3c-cb3c-4385-b59f-d4d6c9a96743/3b7fc7c2f06e48d78bec2125adcc41aa/razorlanguageserver-win-x64-7.0.0-preview.23328.2.zip",
       "installPath": ".razor",
       "platforms": [
         "win32"
       ],
       "architectures": [
         "x86_64"
-      ],
-      "integrity": "BE0001FDAEB075CE4F9148308C09CF97E85DB3BCBFBE0C2A3D70987D8D265047"
```

Comment:
The gulp task to automatically update dependencies wasn't working for me, so I ended up doing this manually (and getting rid of the auto-generated integrity property since it's non-mandatory)

Reply:
+1 to what David said - integrity is useful when its getting downloaded at runtime.  I wouldn't remove it entirel.y

Reply:
Gotcha - I can address this in a follow-up PR as DavidW suggested. Just need to figure out why the script isn't working for me 😅